dist/constants.json

```diff
@@ -8527,157 +8527,157 @@
+        "label": "Newark VA Clinic"
       },
       {
         "value": "561BZ",
-        "label": "JAMES J HOWARD OUTPATIENT CLINIC"
+        "label": "James J. Howard Veterans' Outpatient Clinic"
       },
       {
         "value": "561GA",
-        "label": "HAMILTON CBOC"
+        "label": "Hamilton VA Clinic"
       },
       {
         "value": "561GB",
         "label": "ELIZABETH CBOC"
       },
       {
         "value": "561GD",
-        "label": "HACKENSACK CBOC"
+        "label": "Hackensack VA Clinic"
       },
       {
         "value": "561GE",
-        "label": "JERSEY CITY CBOC"
+        "label": "Jersey City VA Clinic"
       },
       {
         "value": "561GF",
-        "label": "PISCATAWAY CBOC"
+        "label": "Piscataway VA Clinic"
       },
       {
         "value": "561GH",
-        "label": "MORRIS PLAINS CBOC"
+        "label": "Morristown VA Clinic"
       },
       {
         "value": "561GI",
-        "label": "TINTON FALLS CBOC"
+        "label": "Tinton Falls VA Clinic"
       },
       {
         "value": "460HE",
-        "label": "ATLANTIC COUNTY VA CLINIC"
+        "label": "Atlantic County VA Clinic"
       },
       {
         "value": "460HO",
         "label": "SALEM CITY MORC"
       },
       {
         "value": "561GJ",
-        "label": "PATERSON CBOC"
+        "label": "Paterson VA Clinic"
       },
       {
         "value": "642GD",
-        "label": "GLOUCESTER COUNTY VA CLINIC"
+        "label": "Gloucester County VA Clinic"
       },
       {
         "value": "561GK",
-        "label": "SUSSEX VA CLINIC"
+        "label": "Sussex VA Clinic"
       },
       {
         "value": "642GA",
-        "label": "BURLINGTON COUNTY VA CLINIC"
+        "label": "Burlington County VA Clinic"
       },
       {
         "value": "460HG",
-        "label": "CUMBERLAND COUNTY VA CLINIC"
+        "label": "Cumberland County VA Clinic"
       },
       {
         "value": "460GD",
-        "label": "CAPE MAY (WILMINGTON)"
+        "label": "Cape May County VA Clinic"
       },
       {
         "value": "642GF",
-        "label": "CAMDEN VA CLINIC"
+        "label": "Camden VA Clinic"
       },
       {
         "value": "561",
-        "label": "NEW JERSEY HEALTH CARE SYSTEM - EAST ORANGE"
+        "label": "East Orange VA Medical Center"
       }
     ],
     "DE": [
       {
         "value": "460GA",
-        "label": "SUSSEX COUNTY VA CLINIC"
+        "label": "Sussex County VA Clinic"
       },
       {
         "value": "460HK",
-        "label": "WILMINGTON VA MOBILE CLINIC"
+        "label": "Wilmington VA Mobile Clinic"
       },
       {
         "value": "460HL",
         "label": "SMYRNA MORC"
       },
       {
         "value": "460GC",
-        "label": "KENT COUNTY VA CLINIC"
+        "label": "Kent County VA Clinic"
       },
       {
         "value": "460",
-        "label": "WILMINGTON VA MEDICAL CENTER"
+        "label": "Wilmington Regional Benefit Office"
```

Comment:
Let's confirm this name change.

It looks like another instance of us referencing the Benefits Office vs the Medical Facility.